galaxyproject/galaxy
# [22.05] New Datatype: QIIME 2

Initial support for QIIME 2. We've used this successfully in our most recent workshop. 

These changes are used by [q2galaxy](https://github.com/qiime2/q2galaxy), the integration of which is tested by [q2-mystery-stew](https://github.com/qiime2/q2-mystery-stew) (though more focused tests should be written here as well), and will support our [toolshed repo](https://github.com/qiime2/galaxy-tools).

As I'm coming back to this, I am pretty sure there's a bit of lint to clean up. In particular I think the column type conversion between our "QIIME 2 Metadata" file and Galaxy Tabular formats is not at 100% yet. I'd like to see something more robust here.


## How to test the changes?
(Select all options that apply)
- [x] TODO: I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
  1. Install the latest [staging environment for QIIME 2](https://dev.qiime2.org/latest/quickstart/#install-qiime-2-within-a-conda-environment)
  2. Run `q2galaxy template tests some_directory/`
  3. Use planemo to test the resulting tools in the usual way (which should take about an hour or so to run).
 
## License
